scrapbook-tutorial/src/Form.js

```diff
@@ -0,0 +1,61 @@
+import React, { Component } from "react";
+
+class Form extends Component {
+  initialState = {
+    name: "",
+    comment: "",
+  };
+
+  state = this.initialState;
+
+  handleChange = (event) => {
+    const { name, value } = event.target;
+
+    this.setState({
+      [name]: value,
+    });
+  };
+
+  render() {
+    const { name, comment } = this.state;
+
+    return (
+      <form>
+        <ul class="wrapper">
+          <li class="form-row">
```

Comment:
It might be nice to have each list item in it's own component.  Would make this easier to read and if your project grew and you had more forms it would make it easy to use the same components in different places.

Reply:
Good spot... changed to `commentItem`, `commentItem-name` and `commentItem-name`